deploy-manage/deploy/cloud-on-k8s/pod-disruption-budget.md

```diff
@@ -12,9 +12,45 @@ products:
 
 A [Pod Disruption Budget](https://kubernetes.io/docs/tasks/run-application/configure-pdb/) (PDB) allows you to limit the disruption to your application when its pods need to be rescheduled for some reason such as upgrades or routine maintenance work on the Kubernetes nodes.
 
-ECK manages a default PDB per {{es}} resource. It allows one {{es}} Pod to be taken down, as long as the cluster has a `green` health. Single-node clusters are not considered highly available and can always be disrupted.
+ECK manages either a single default PDB, or multiple PDBs per {{es}} resource according to the license available.
 
-In the {{es}} specification, you can change the default behavior as follows:
+:::{note}
+In ECK 3.1 and earlier, all clusters follow the [non-enterprise behavior](#non-enterprise-licensed-customers), regardless of license type.
+:::
+
+## Enterprise licensed customers
+```{applies_to}
+deployment:
+  eck: ga 3.2
+```
+
+In {{eck}} clusters licensed with an enterprise license, a separate PDB is created for each type of nodeSet defined in the manifest allowing upgrade or maintenance operations to be more quickly executed. The PDBs allow one {{es}} Pod per nodeSet to simultaneously be taken down as long as the cluster has the health defined in the following table:
+
+| Role | Cluster health required | Notes |
+|------|------------------------|--------|
+| master | Yellow |  |
+| data | Green | All Data roles are grouped together into a single PDB, except for data_frozen. |
+| data_frozen | Yellow | Since data_frozen nodes are essentially stateless, managing searchable snapshots when compared to other data node types, additional disruptions are allowed. |
```

Comment:
```suggestion
| data_frozen | Yellow | Since frozen data tier nodes only host partially mounted indices backed by searchable snapshots additional disruptions are allowed. |
```
I think the main argument is not that they are stateless. They kind of are but even for a stateless node that local cache is something to protect for availability. I think the main argument for additional disruptions here is that there is only one "replica" on the frozen tier anyway so unavailability is to be expected.